frontend/src/routes/Schedule/index.css

```diff
@@ -80,3 +79,9 @@
   color: #333;
   margin-top: 15px;
 }
+
+@media screen and (max-width: 768px) {
+  .game-cards-container {
+    padding: unset;
+  }
+}
```

Comment:
Please move this into the existing .games-cards-container class above.

You will no longer need to specify `.game-cards-container` once you do so and can directly change the padding inside

Reply:
done